src/Kunstmaan/UtilitiesBundle/Tests/Helper/SlugifierTest.php

```diff
@@ -40,7 +40,8 @@ public function getSlugifyData()
             array('een titel met spaties', '', 'een-titel-met-spaties'),
             array('à partir d\'aujourd\'hui', null, 'a-partir-d-aujourd-hui'),
             array('CaPs ShOulD be LoweRCasEd', null, 'caps-should-be-lowercased'),
-            array('áàäåéèëíìïóòöúùüñßæ', null, 'aaaaeeeiiiooouuunssae'),
+            array('áàäåéèëíìïóòöúùüñßæ', null, 'aaaaeeeiiiooouuunssae'),,
```

Comment:
Of course... sorry for stupid mistake ... this was second pull request, because first I did in KunstmaanUtilitiesBundle... so this one I didn't test...